.github/workflows/release.yml

```diff
@@ -347,18 +319,31 @@ jobs:
           echo "Metabase OSS: image is going to be pushed to ${{ github.repository_owner }}/metabase"
           echo "DOCKERHUB_REPO=${{ github.repository_owner }}/metabase" >> $GITHUB_ENV
         fi
-
+    - uses: actions/download-artifact@v4
+      name: Retrieve previously downloaded Uberjar
+      with:
+        name: metabase-${{ matrix.edition }}-uberjar
+    - name: Move the Uberjar to the context dir
+      run: mv ./metabase.jar bin/docker/.
     - name: Login to Docker Hub
       uses: docker/login-action@v2
       with:
         username: ${{ secrets.DOCKERHUB_RELEASE_USERNAME }}
         password: ${{ secrets.DOCKERHUB_RELEASE_TOKEN }}
-    - name: Retag and push container image to Docker Hub
-      run: |
-        echo "Pushing ${{ steps.canonical_version.outputs.result }} to ${{ env.DOCKERHUB_REPO }} ..."
-        docker tag localhost:5000/local-metabase:${{ steps.canonical_version.outputs.result }} ${{ env.DOCKERHUB_REPO }}:${{ steps.canonical_version.outputs.result }}
-        docker push ${{ env.DOCKERHUB_REPO }}:${{ steps.canonical_version.outputs.result }}
-        echo "Finished!"
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v3
+    - name: Build ${{ matrix.edition }} container
+      uses: docker/build-push-action@v6
+      with:
+        context: bin/docker/.
+        platforms: linux/amd64,linux/arm64
+        network: host
+        tags: ${{ env.DOCKERHUB_REPO }}:${{ steps.canonical_version.outputs.result }}
```

Comment:
This is also a change. Instead of building and "pushing" to a local registry so we can test before pushing to dockerhub, we just build and push to dockerhub directly, which speeds things up a bit, and simplifies the multi-arch push.

FWIW, I have never seen the post-build test fail. We also have another post-push test to catch if we somehow push a broken image.

Reply:
What's the process in the unlikely event the post-push test fails? Just push a new image and delete the old broken one?

Reply:
I like having a test inside the script, docker can corrupt the image so break CI if something is wrong